FlexateWebApi/Controllers/OfficesController.cs

```diff
@@ -91,7 +89,7 @@ public async Task<ActionResult> Create([FromBody] CreateOfficeDto officeDto, Can
                 return BadRequest();
             }
 
-            return Created($"api/offices/{id}", id);
```

Comment:
this comment is for line 83 - it's good practice to use post instead of create in controller :)